int32 timeTag
geometry_msgs/Quaternion rotation1
float64 translation
geometry_msgs/Quaternion rotation2
